crates/fluvio-controlplane-metadata/src/managed_connector/k8/spec.rs

```diff
@@ -0,0 +1,78 @@
+//!
```

Comment:
This comment refers to SPU group. Is this for rust docs? If not, it should be removed.